Dockerfile

```diff
@@ -1,3 +1,5 @@
+ARG RUNTIME_IMAGE=alpine:3.15
+
```

Comment:
This `ARG` has to be at the top, otherwise the docker daemon does not known what to do with `FROM ${RUNTIME_IMAGE}`

Reply:
Might be worth adding a comment with this context into the file directly

Reply:
Good idea, I'll add it over the new years